ldk/javascript/examples/self-test-loop/src/tests/whisper.ts

```diff
@@ -0,0 +1,700 @@
+import { clipboard, whisper } from '@oliveai/ldk';
```

Comment:
Is this diff not showing these tests moved from `self-test-loop/src/tests/index.ts` or were they not removed?

Reply:
Well, looks like merging from main brought them back. Let me fix it.

Reply:
Cleaned up